pkg/pulumiyaml/run.go

```diff
@@ -1507,6 +1503,15 @@ func (e *programEvaluator) evaluatePropertyAccess(expr ast.Expr, access *ast.Pro
 		receiver = p
 	} else if v, ok := e.variables[resourceName]; ok {
 		receiver = v
+	} else if v, ok := e.config[stripConfigNamespace(resourceName)]; ok {
+		receiver = v
+		e.addDiag(&syntax.Diagnostic{
+			Diagnostic: hcl.Diagnostic{
+				Severity: hcl.DiagWarning,
+				Summary:  "Project config keys should not be namespaced;",
+				Detail:   fmt.Sprintf("Use %s instead of %s", stripConfigNamespace(resourceName), resourceName),
+			},
+		})
```

Comment:
I don't think this is enforced by Pulumi for other languages - what's the motivation for us doing that here?

Reply:
No firm reason – we can remove the warn to be consistent then